src/udiskslinuxpartitiontable.c

```diff
@@ -544,7 +544,7 @@ udisks_linux_partition_table_handle_create_partition (UDisksPartitionTable   *ta
      the new partition without getting change event for the disks after the
      last add event and this breaks the "Partitions" property on the
      "PartitionTable" interface. */
-  udisks_linux_block_object_trigger_uevent (UDISKS_LINUX_BLOCK_OBJECT (object));
+  udisks_linux_block_object_trigger_uevent_sync (UDISKS_LINUX_BLOCK_OBJECT (object), 10);
```

Comment:
These timeouts look a little random. Some functions have 5, some 10 or 15 seconds. Can we unify this or document somewhere the reason for different timeouts?

Reply:
Right, they are random. At least I don't remember and can't find any reason for that.

Added a `UDISKS_DEFAULT_WAIT_TIMEOUT` constant (20 seconds) that is now used in (almost) all wait functions.